src/formatDate.js

```diff
@@ -8,7 +8,84 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromFormatObject = arrayInObject(fromFormat);
```

Comment:
The function `arrayInObject` is not defined in the provided code. Ensure that this function exists and is correctly implemented to transform the `fromFormat` and `toFormat` arrays into objects.